### PR TITLE
:tada: Setting up webpack for smooth development

### PR DESCRIPTION
**Pull request by PranjalAgni**
Setting up webpack for smooth development

### CODE REVIEW

**Comment by PranjalAgni** [comment]
Need some help here getting a weird issue, `webpack` does not refresh browser  after compiling the new changes.

**Comment by RocktimSaikia** [comment]
@PranjalAgni Hi , I will run some tests on your branch and let you know what's going on as soon as i'm done !

**Comment by PranjalAgni** [comment]
Hey I have fixed the hot-reloading webpack issue which was happening you can have a look.

File: package.json [hunk 1]
```diff
@@ -5,17 +5,22 @@
...
+    "start": "nodemon index.js -e ejs"
   },
   "author": "Rocktim Saikia",
   "license": "MIT",
   "dependencies": {
     "axios": "^0.19.0",
     "ejs": "^2.6.1",
-    "express": "^4.16.4"
+    "express": "^4.16.4",
+    "webpack": "^4.41.0"
```
Comment (RocktimSaikia) [inline package.json]: @PranjalAgni can you remove the webpack dependencies from the  as we don't need it anymore !

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,4 @@
 node_modules
-package-lock.json
\ No newline at end of file
+package-lock.json
+.vscode
+dist
```
Comment (RocktimSaikia) [inline .gitignore]: and remove the dist from `gitignore` too as no bundle is being created by any of the packages

File: package.json (hunk 1 above)
Comment (PranjalAgni) [reply package.json]: Yup :)

File: .gitignore (hunk 2 above)
Comment (PranjalAgni) [reply .gitignore]: Yup :)

**Comment by PranjalAgni** [comment]
Done 💯

**Pull request closed without merging**